data/pipeline/scripts/locations/parse_congress_districts.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# takes congress_members.html from stdin, and writes to stdout a CSV of Colorado
+# congress members.
+# each row includes the district number, member name, affiliation, and website
+# (website is derived from the member's name)
+
+# use hq to extract table with CSS class name "library-table"
+# then use jq (in several steps; optimize later?) to extract the data, annotate
+# with some computed fields, and format it as CSV
+hq 'array {
+    //table//tr ->
```

Comment:
Didn't know about the hq command, this is very useful. I think maybe you could insert a `tbody` here to avoid also selecting the header row. Can't tell if any of the code below this filters out empty member/affiliation/state/district though.

Reply:
Good call; I changed it to that after trying it out, thanks. It does implicitly get filtered out due to the `.state == "Colorado (CO)"` clause in the `jq` invocation later on, but still, better not to parse it at all.

FWIW `hq` isn't my favorite tool, mostly because the documentation is kind of all over the place. Regardless, I tried a few on this somewhat messy page and it was the one that worked with the least complaints. If you're curious about it, though, here's the GitHub link to it: https://github.com/rbwinslow/hq.